release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -935,6 +935,18 @@ The deprecated `dhclient` binary has been removed from {op-system}. Starting wit
 
 *Node*
 
+* Previously, in containers, CRI-O did not create a symbolic link from `/proc/mounts` file to the `/etc/mtab` file. As a consequence, the user could not view the list of the mounted devices in the container's `/etc/mtab` file. CRI-O now adds the symbolic link. As a result, users can view the container's mounted devices.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1868221[*BZ#1868221*])
+
+* Previously, if pods were deleted quickly after creation, the kubelet might not clean up the pods properly. This resulted in pods being stuck in terminating state could impact availability of upgrades. This fix improves the pod lifecycle logic to avoid this problem.
```

Comment:
Does the second sentence read weird to you?

Reply:
yes, will see about fixing that